### ISSUE TITLE
OpenScienceKE

### ISSUE THREAD

**Issue by kipkurui** [MozillaFestival/open-leaders-6#30]
Project Lead: @kipkurui

Mentor: @felipez3r0

Welcome to OL6, Cohort E! This issue will be used to track your project and progress during the program. Please use this checklist over the next few weeks as you start Open Leadership Training :tada:.

***
Before Week 1 (Sept 13): Your first mentorship call
- [x] Complete the OLF self-assessment ([online](https://mzl.la/olf-online-assessment), [printable](https://mzl.la/olf-self-assessment)). If you're a group, each teammate should complete this assessment individually. This is here to help you set your own personal goals during the program. No need to share your results, but be ready to share your thoughts with your mentor.
- [x] Make sure you know when and how you'll be meeting with your mentor.

Before Week 2 (Sept 20): First Cohort Call (Open by Design)
- [x] Complete Assignment ['Write Your Vision or Mission Statement'](https://mozilla.github.io/open-leadership-training-series/articles/introduction-to-open-leadership/stating-your-project-vision/). Leave a comment on this issue with your draft vision statement & be ready to share this on the call.
- [x] Watch ['Mozilla & Working Open'](https://youtu.be/quKdaqlR_9w).
- [x] Check the [Syllabus](https://mzl.la/ol6-syllabus-culture-track) for notes and connection info for all the cohort calls.

Before Week 3 (Sept 27): Mentorship call
- [x] Look up two other projects and comment on their issues with feedback on their vision statement.
- [x] Complete your Open Canvas ([instructions](https://mozilla.github.io/open-leadership-training-series/articles/opening-your-project/develop-an-open-project-strategy-with-open-canvas/), [canvas](https://goo.gl/to6PYn)). Comment on this issue with a link to your canvas.
- [x] [Start your Roadmap](https://mozilla.github.io/open-leadership-training-series/articles/opening-your-project/start-your-project-roadmap/). Comment on this issue with your draft Roadmap.

Before Week 4 (Oct 4): Cohort Call (Build for Understanding)
- [x] Look up two other projects and comment on their issues with feedback on their open canvas.
- [x] [Pick an open license](https://mozilla.github.io/open-leadership-training-series/articles/get-your-project-online/sharing-your-work-in-the-open/) for the work you're doing during the program.
- [x] Use your canvas to [start writing a README](https://mozilla.github.io/open-leadership-training-series/articles/opening-your-project/write-a-great-project-readme/), or landing page, for your project. Link to your README in a comment on this issue.

This issue is here to help you keep track of work during the first month of the program. Please refer to the [OL6 Syllabus](https://mzl.la/ol6-syllabus-culture-track) for more detailed weekly notes and assignments past week 4.

**Comment by bashybaranaba** [comment]
I live in Kenya, and I believe the Bioinformatics industry and other industries in Kenya should promote open and collaborative environments.I am keen to see the progress of your project.

**Comment by kipkurui** [comment]
> I live in Kenya, and I believe the Bioinformatics industry and other industries in Kenya should promote open and collaborative environments.I am keen to see the progress of your project.

Indeed! Thanks for your interest. I look forward to meeting you sometime.

**Comment by kipkurui** [comment]
Here is a link to my [draft open canvas](https://docs.google.com/presentation/d/1IlMVBqeYuzviExgekyQqXuKhY_opB7Sy1egU38uvDc4/edit?usp=sharing) @felipez3r0.

**Comment by kipkurui** [comment]
Here is a working [draft of OpenScienceKE roadmap](https://www.evernote.com/shard/s96/sh/073a8ac7-ac8b-470c-b79b-58dfd651003c/542e03785b77fb284dbcc0057162521f)

**Comment by mlbonatelli** [comment]
> **Vision Statement**
> I am working with Bioinformatics students and researchers to promote open, collaborative and reproducible Bioinformatics Research in Nairobi, Kenya so that we can we can build a community of Bioinformaticians who can collaborate and spread the open science message.

Hey @kipkurui, really nice statement! Any Bioinformatic Research area in particular?

**Comment by kipkurui** [comment]
> > **Vision Statement**
> > I am working with Bioinformatics students and researchers to promote open, collaborative and reproducible Bioinformatics Research in Nairobi, Kenya so that we can we can build a community of Bioinformaticians who can collaborate and spread the open science message.
> 
> Hey @kipkurui, really nice statement! Any Bioinformatic Research area in particular?

Thank you @mlbonatelli. At this stage, we are not focusing on any area of Bioinformatics. We are equipping participants with Open Science tools: Git and GitHub for collaboration, Jupyter and R notebooks for reproducibility, Linux and Markdown. 

We'll move later on to Reproducible Bioinformatics Pipelines (Snakemake, Common Workflow Language, etc) and environment management (bioconda), and Containerization (Docker and Singularity).

**Comment by bmuenzenmeyer** [comment]
I've read through your open canvas and learned a lot. It seems like a daunting amount of effort, but I applaud your boldness and desire to make positive change. I love the idea of how enabling your efforts will be for others.

❤️

**Comment by mlbonatelli** [comment]
> > > **Vision Statement**
> > > I am working with Bioinformatics students and researchers to promote open, collaborative and reproducible Bioinformatics Research in Nairobi, Kenya so that we can we can build a community of Bioinformaticians who can collaborate and spread the open science message.
> > 
> > 
> > Hey @kipkurui, really nice statement! Any Bioinformatic Research area in particular?
> 
> Thank you @mlbonatelli. At this stage, we are not focusing on any area of Bioinformatics. We are equipping participants with Open Science tools: Git and GitHub for collaboration, Jupyter and R notebooks for reproducibility, Linux and Markdown.
> 
> We'll move later on to Reproducible Bioinformatics Pipelines (Snakemake, Common Workflow Language, etc) and environment management (bioconda), and Containerization (Docker and Singularity).

I agree with @bmuenzenmeyer ! And I wish you a good luck with your project!

**Comment by kipkurui** [comment]
Thank you @mlbonatelli @bmuenzenmeyer. Yes, it is quite a daunting task, but the progress we have made so far is very encouraging.

**Comment by nerantzis** [comment]
Very interesting project!

**Comment by kipkurui** [comment]
Thank you @nerantzis
